.github/workflows/api-cd.yml

```
@@ -0,0 +1,87 @@
+          cat > .env << EOF
+          DOCKER_REGISTRY=${{ secrets.DOCKER_REGISTRY }}
+          MYSQL_USER=${{ secrets.MYSQL_USER }}
+          MYSQL_PASSWORD=${{ secrets.MYSQL_PASSWORD }}
+          MYSQL_ROOT_PASSWORD=${{ secrets.MYSQL_ROOT_PASSWORD }}
+          MYSQL_HOST=${{ secrets.MYSQL_HOST }}
+          MYSQL_PORT=${{ secrets.MYSQL_PORT }}
+          MYSQL_DATABASE=${{ secrets.MYSQL_DATABASE }}
+          EOF
+
+      - name: 🚀 Deploy
+        run: |          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)